changelog/changes/2023-07-05-drop-niftycloud-and-interoute.md

```diff
@@ -0,0 +1 @@
+- :warning: Dropped support for niftycloud and interoute OEMs. For interoute we haven't been generating the images for some time already. (TODO) :warning:
```

Comment:
```suggestion
- :warning: Dropped support for niftycloud and interoute OEM images. For interoute we haven't been generating the images for some time already. (TODO) :warning:
```

Reply:
Or without `OEM images`, either both words or none?

Reply:
Went with the "none" variant.